CONTRIBUTING.md

```diff
@@ -1,78 +0,0 @@
-# Contributing
```

Comment:
@seanmarcia wanted to double check with you if you think this is ok to remove this document and keep it in README.md